aip/general/0151.md

```diff
@@ -59,6 +59,22 @@ rpc CreateBook(CreateBookRequest) returns (google.longrunning.Operation) {
 - APIs with messages that return `Operation` **must** implement the
   [`Operations`][lro] service. Individual APIs **must not** define their own
   interfaces for long-running operations to avoid non-uniformity.
+- If an RPC supports a [validate-only mode](aip-163), the response to a
+  validation request **must** be one of the following:
+  - A successful response with an `Operation` which is already complete, with
+    the `done` field set to `true`, and a valid (but potentially empty) response
+    message in the `result` field, wrapped in a `google.protobuf.Any` message.
```

Comment:
```suggestion
    message in the `response` field, wrapped in a `google.protobuf.Any` message.
```

`result` is the `oneof` name, the field itself is `response`.

Reply:
Doh, fixed, thanks.